tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature

```diff
@@ -11,21 +11,6 @@ Feature: accept/decline shares coming from internal users
       | Brian    |
     And user "Brian" has logged in using the webUI
 
-  @issue-ocis-1950
-  Scenario: reject a share that you received as user and as group member
```

Comment:
This test doesn't work in that form anymore because oCIS now sends 2 shares when one resource is being shared with a user directly as well as via group.

Could be refactored, but since it's an acceptance test and we won't have the concept of "declining" shares in the future I guess we can just let the test die.